parsec/parsec_mpi_funnelled.c

```diff
@@ -1393,11 +1393,13 @@ mpi_no_thread_enable(parsec_comm_engine_t *ce)
         MPI_Info_set(no_order, "mpi_assert_allow_overtaking", "true");
         MPI_Comm_dup_with_info((MPI_Comm) context->comm_ctx, no_order, &parsec_ce_mpi_comm);
         MPI_Info_free(&no_order);
-    } else {
-        MPI_Comm_dup((MPI_Comm) context->comm_ctx, &parsec_ce_mpi_comm);
+        /* There is no need to enable overtake for the AM communicator */
     }
-    /* There is no need to enable overtake for the AM communicator */
+    else
```

Comment:
Why do we even have the conditional here? It's an info key, just set and forget it. No downside if it's not supported.

Reply:
we could also dup_with_info with an empty info

Reply:
Why? Just set the key and be done with it. Complexity leads to errors like this.